Root_AdminMenu.php

```diff
@@ -209,7 +209,7 @@ public function options() {
 		/*
 		 * Hidden pages.
 		 */
-		if ( isset( $_REQUEST['w3tc_dbcluster_config'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification
+		if ( isset( Util_Request::get_string( 'w3tc_dbcluster_config' ) ) ) { // phpcs:ignore WordPress.Security.NonceVerification
```

Comment:
Revert to use ` isset( $_REQUEST[''w3tc_dbcluster_config] )` or use `! empty()`.

Reply:
Changed to ! empty.